db/seeds/demo/demo_students.seeds.rb

```diff
@@ -31,11 +31,11 @@
   result = StarResult.new(FakeStarResult.data)
   result.update_attributes(student_id: student.id)
   result.save
-  rand(0..3).times do
-    discipline_incident = DisciplineIncident.new(FakeDisciplineIncident.data)
-    discipline_incident.student_id = student.id
-    discipline_incident.save
-  end
+  # rand(0..3).times do
```

Comment:
i just did this so it was easier to find zero case situations.  if a student has an incident in a previous year it will still be charted.